persistence/src/main/java/edu/unc/lib/dl/persist/services/move/MoveObjectsJob.java

```diff
@@ -141,10 +143,63 @@ private void moveObject(PID objPid) {
         PID sourcePid = moveContent.getParent().getPid();
         addPidToSource(objPid, sourcePid);
 
+        // Write a premis event if object moves between admin units
+        adminUnitMove(objPid, moveContent);
+
         // Add the object to its destination, which clears the previous parent as well
         destContainer.addMember(moveContent);
     }
 
+    private void adminUnitMove(PID sourcePid, ContentObject moveObj) {
+        HashMap<String, String> destContainerInfo = getContainerInfo(destinationPid, 1);
```

Comment:
```suggestion
        Map<String, String> destContainerInfo = getContainerInfo(destinationPid, 1);
```
Use interface rather than implementation